openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -55,7 +58,7 @@ def setUpClass(cls):
         # pylint: disable=super-method-not-called
         with super(TestCourseOutlinePage, cls).setUpClassAndTestData():
             cls.courses = []
-            course = CourseFactory.create()
+            course = CourseFactory.create(self_paced=True)
```

Comment:
Just a general comment. Might want to be careful with these as it could potentially cause some stuff to fail due to implicit differences between instructor paced and self paced courses.

Reply:
so i changed this stuff in a couple files.  i hear what you're saying and it came to mind, but the current tests don't test for any differences between instructor paced and self pace, and given the nature of our project, seems like self paced may be the "go to"